prov/usnic/src/usdf_cq.c

```diff
@@ -428,6 +515,94 @@ usdf_cq_copy_soft_entry(void *dst, const struct usdf_cq_soft_entry *src,
 	return FI_SUCCESS;
 }
 
+static ssize_t
+usdf_cq_sread_common_soft(struct fid_cq *fcq, void *buf, size_t count, const void *cond,
```

Comment:
looks like you've resolved this note but github thinks it still applies to the updated diff (no need to do anything, just making it clear to my future self that this is resolved)